index.html

```diff
@@ -161,6 +163,31 @@ <h3 class="inner">skills</h3>
             </ul>
         </div>
     </div>
+</section>
+<section class="about-me">
+  <img src="images/Ellipse 3.svg" class="img1">
+  <img src="images/Rectangle 4.svg" class="img2">
+  <img src="images/Ellipse 12.svg" class="img3">
+  <img src="images/Ellipse 5.svg" class="img4">
+  <img src="images/Rectangle 7.svg" class="img5">
+  <img src="images/Shape.svg"class= "img6">
```

Comment:
- [ ] These images are not supposed to be here, Kindly make sure you download the entire background image which contains all these little shapes and set it as background.